man/systemd.time.xml

```diff
@@ -123,7 +123,11 @@
 
     <para>When parsing, systemd will accept a similar syntax, but expects no timezone specification, unless it is given
     as the literal string <literal>UTC</literal> (for the UTC timezone) or is specified to be the locally configured
-    timezone. Other timezones than the local and UTC are not supported. The weekday specification is optional, but when
+    timezone. Other timezones than the local and UTC are not supported. Starting from v235 it is possible to specify
```

Comment:
We tend to document current state of affairs, so “Starting from v235” should be dropped.
It adds little information, duplicated (we have NEWS file for that) and quickly grows into mess of historical information mixed with useful things.

Reply:
Agree, changed. Thanks.